### ISSUE TITLE
Under cygwin netmask filled incorrectly

### ISSUE THREAD

**Issue by pyhedgehog** [smurn/ifaddr#1]
## Original error
```console
$ uname -a
CYGWIN_NT-6.1 MICHAELD04 2.4.1(0.293/5/3) 2016-01-24 11:26 x86_64 Cygwin
$ python2.7 -c 'import ifaddr;print ifaddr.get_adapters()'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/ifaddr/_posix.py", line 65, in get_adapters
    prefixlen = ipaddress.IPv4Network(unicode('0.0.0.0/' + netmask)).prefixlen
TypeError: cannot concatenate 'str' and 'NoneType' objects
```
### test1.py
```python
import os, ifaddr, ctypes, socket
assert os.name == 'posix'
addr = ctypes.POINTER(ifaddr._posix.ifaddrs)()
retval = ifaddr._posix.libc.getifaddrs(ctypes.byref(addr))
assert retval==0
while addr and addr[0].ifa_addr[0].sa_familiy != socket.AF_INET6:
    addr = addr[0].ifa_next
assert addr
print 'addr[0].ifa_addr[0].sa_familiy =', addr[0].ifa_addr[0].sa_familiy
print 'addr[0].ifa_netmask[0].sa_familiy =', addr[0].ifa_netmask[0].sa_familiy
```
### Test output
```console
$ python2.7 test1.py
addr[0].ifa_addr[0].sa_familiy = 23
addr[0].ifa_netmask[0].sa_familiy = 0
```

*PS: I'll post PR soon.*

**Comment by smurn** [comment]
Thanks!

Looks like a reasonable workaround to me. I've never tested it on cygwin. Thanks for making it work there too!

I'll release a new version on pypi shortly (including the memory leak fix).

-Stefan

**Comment by smurn** [comment]
FYI, Released version 0.1.3.

**Comment by pyhedgehog** [comment]
Thanks.
